subsys/usb/device/class/cdc_acm.c

```diff
@@ -535,6 +549,16 @@ static int cdc_acm_fifo_read(const struct device *dev, uint8_t *rx_data,
 		dev_data->rx_ready = false;
 	}
 
+	if(dev_data->receive_paused == true) {
+		if(ring_buf_space_get(dev_data->rx_ringbuf) > CDC_ACM_BUFFER_SIZE){
```

Comment:
You are right, I'll change the comparation to greater then or equal. There is no reason to use smaller ring buffer size then CDC_ACM_BUFFER_SIZE - this will result in data loss for every transfer greater then ring buffer size.